doc/lua_api.txt

```diff
@@ -212,6 +212,8 @@ Aliases can be added by using `minetest.register_alias(name, convert_to)`.
 This will make Minetest to convert things called name to things called
 `convert_to`.
 
+If an item called name exists, it will be unregistered.
```

Comment:
`name` -> \`name\`

`register_alias` does not unregister an item with `name`. It's what `register_alias_force` does.